index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
```

Comment:
- [ ] Kindly 😊 try to use the general style for the page to remove the default margin and padding from the elements this will make the design great 😃

> This is a general style kindly try to put them in the CSS file ⤵️

```code
*{
padding: 0;
margin: 0;
box-sizing: border-box;
}
```

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220179127-02fd4a1b-0b02-48a6-a6e2-62010d58900a.png)   |   ![image](https://user-images.githubusercontent.com/83360636/220179264-a832874c-600b-4176-90db-7cf25b8a0a7d.png)   |

Reply:
Hi @osamaashraf6 

I have noticed why you found issue my code but while there is no any issue from my side. Actually, you have checked my code with adding below general style for CSS and that is why everything in my style shows for you messed up. May I ask you to check my code without below  general CSS style as I did not considered while coding my CSS style to see if everything is OK.
*{
padding: 0;
margin: 0;
box-sizing: border-box;
}